packages/bpk-component-accordion/src/BpkAccordionItem-test.js

```diff
@@ -74,7 +74,7 @@ describe('BpkAccordionItem', () => {
     expect(tree).toMatchSnapshot();
   });
 
-  it('should render correctly with "tagName" prop set', () => {
+  it('should render correctly with "bold" prop set', () => {
```

Comment:
Good spot!

Reply:
Raised by eslint also, I think that because of the new `eslint-plugin-jest` added in https://github.com/Skyscanner/eslint-config-skyscanner/pull/30